### PR TITLE
Removing redundant renewal btn

### PR DESCRIPTION
**Pull request by matzew**
Done for AGPUSH-1503

### CODE REVIEW

**Comment by matzew** [comment]
@thradec @lfryc mind to review/merge ?

File: admin-ui/app/components/app-detail/include/variantDetail.html [hunk 1]
```diff
@@ -52,7 +52,7 @@
   <dt>Type:</dt>
   <dd>
     Microsoft Push Notification Service<br />
-    <button class="btn btn-sm btn-default" ng-click="variants.renewVariantSecret( variant )"><span class="pficon pficon-refresh"></span> Renew Variant Secret</button>
+<!-- <button class="btn btn-sm btn-default" ng-click="variants.renewVariantSecret( variant )"><span class="pficon pficon-refresh"></span> Renew Variant Secret</button> -->
```
Comment (lfryc) [inline admin-ui/app/components/app-detail/include/variantDetail.html]: why commenting, shouldn't we just remove it?

**Comment by matzew** [comment]
@lfryc addressed it - and ys, got rid of the commented code :cactus:

**Comment by matzew** [comment]
@lfryc ok to merge ?

**Comment by lfryc** [comment]
testing now...

**Comment by matzew** [comment]
@lfryc ok to merge ?

**Comment by lfryc** [comment]
code-wise it looks fine, but I wasn't able to test it locally yet, the DB setup process is now too paintful

**Pull request merged**